maintainers/default.nix

```
@@ -0,0 +1,25 @@
+{
+  stdenvNoCC,
+  makeWrapper,
+  python3,
+}:
+
+{
+  render_templates = stdenvNoCC.mkDerivation {
+    name = "render_templates";
+
+    src = self;
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/bin
+      makeWrapper ${python3.interpreter} $out/bin/render_templates \
+      --add-flags "${./scripts/render_templates.py}"
+
+      runHook postInstall
+    '';
+  };
+}
```

I tested this following the suggestion by @fgaz https://github.com/nix-community/nur-packages-template/pull/109/files#r2202868549
```suggestion
{
  jinja2-cli,
  writeShellScriptBin,
}:
{
  render_templates  = writeShellScriptBin "render_templates" ''
    find . -name "*.j2" -print0 | while read -d $'\0' file; do
      ${jinja2-cli}/bin/jinja2 "$file" config.yml --format=yml > "''${file%.j2}"
    done
  '';
}
```